ElixirSeattle/xebow
# Use mix upload command from the latest nerves_firmware_ssh

Related to #55 

This PR updates us to the cutting edge of `nerves_firmware_ssh` where `mix upload` is being introduced as an official command, and `upload.sh` will no longer be required. I've made a couple other tweaks to our `mix.exs` file to accommodate for some xebow-specific conveniences.

I plan on keeping this an open PR until `mix upload` makes it into a release, unless you all feel differently.

## Steps for testing

- Check out this branch
  - `refactor/nerves_firmware_ssh-upload`
- Update dependencies
  - `MIX_TARGET=keybow mix deps.get`
- Build the firmware
  - `mix firmware`
- Upload the firmware
  - `mix upload`
